python_modules/dagster/dagster/_core/pipes/context.py

```diff
@@ -124,12 +129,8 @@ def handle_message(self, message: PipesMessage) -> None:
         if self._received_closed_msg:
             self._context.log.warn(f"[pipes] unexpected message received after closed: `{message}`")
 
-        if not self._received_any_msg:
-            self._received_any_msg = True
-            self._context.log.info("[pipes] external process successfully opened dagster pipes.")
-
         if message["method"] == "opened":
-            pass
+            self._handle_opened(message["params"])  # type: ignore
```

Comment:
why type ignore?

Reply:
It's explained in the comment above `handle_message`-- following the pattern for all existing messages:

```
# Type ignores because we currently validate in individual handlers
```

Reply:
Would be nice to not require this but not going to block